compatibility/BUILD

```diff
@@ -52,12 +52,18 @@ config_setting(
     for sdk_version in sdk_versions
 ]
 
+# Instead of testing the full cartesian product of all SDK versions with
+# all platform (~= Sandbox/JSON API) versions, we pin the SDK version to be
+# the latest one. This gives us a reasonable feedback with regards to
+# maintaining backwards-compatibility without causing the test runs to grow
```

Comment:
they grow quadratically not exponentially :slightly_smiling_face:

Reply:
Of course, thanks.

Reply:
Push forced and fixed in https://github.com/digital-asset/daml/pull/8893/commits/599a135d35318d22e810e6bc86026621b5746056

I changed the PR description to match the commit message.